### ISSUE TITLE
ENHANCEMENT: prop autofill

### ISSUE THREAD

**Issue by CreaturesInUnitards** [tomblachut/svelte-intellij#12]
It would be great if a component's exported parameters would be included in the HTML autocompletion. So f.e. if this is my component:

```html
// MyComponent.svelte
<script>
  export let foo
  export let bar
</script>
```

When I type 
```html
<MyCom
```
into a consuming component and accept the autocompletion, I would then get
```html
<MyComponent foo="" bar="" />
```

Some related nice-to-haves:
1. the completions behave as Live Templates, with tab stops in between the quotes, e.g. `foo="$foo$"`
2. the completions lay out any existing slots, or self-close the tag if there are none

**Comment by tomblachut** [comment]
Depends on #5.

A fine suggestion, yet some people would prefer simply display completion dropdown after inserting consecutive props. Similar to handling of arguments to a function. One could also scan local identifiers and suggest for example
```
<MyComponent {foo} {bar} />
```

**Comment by tomblachut** [comment]
Hmm I'm gonna close this issue. @unlocomqx handled actual title. 
Outstanding issue is 
> the completions lay out any existing slots, or self-close the tag if there are none
Which is another thing altogether - a nice idea of course!
